caringcaribou/modules/uds.py

```diff
@@ -837,7 +842,7 @@ def __auto_wrapper(args):
     try:
         arb_id_pairs = uds_discovery(min_id, max_id, blacklist,
                                      auto_blacklist_duration,
-                                     delay, verify, print_results)
+                                     delay, verify, padding, print_results)
```

Comment:
`padding` is undefined here, causing the whole `__auto_wrapper` function to break:

```
$ cc.py -i vcan0 uds auto

-------------------
CARING CARIBOU v0.4
-------------------

Loading module 'uds'


Traceback (most recent call last):
  File "/usr/local/bin/cc.py", line 33, in <module>
    sys.exit(load_entry_point('caringcaribou==0.4', 'console_scripts', 'cc.py')())
  File "/usr/local/lib/python3.10/dist-packages/caringcaribou-0.4-py3.10.egg/caringcaribou/caringcaribou.py", line 125, in main
    cc_mod.module_main(args.module_args)
  File "/usr/local/lib/python3.10/dist-packages/caringcaribou-0.4-py3.10.egg/caringcaribou/modules/uds.py", line 1347, in module_main
    args.func(args)
  File "/usr/local/lib/python3.10/dist-packages/caringcaribou-0.4-py3.10.egg/caringcaribou/modules/uds.py", line 845, in __auto_wrapper
    delay, verify, padding, print_results)
NameError: name 'padding' is not defined
```